vulcan/_vulcan.py

```diff
@@ -144,3 +144,15 @@ def get_messages(self, date_from=None, date_to=None):
             :class:`vulcan._message.Message`
         """
         return Message.get(self._api, date_from, date_to)
+    def send_message(self, title, content, recipient_id, recipients):
```

Comment:
Remove parameters `recipient_id` and `recipients` and instead of it, add `teachers` argument, that'll accept list of `Teacher` objects.